Runtime/Behaviours/CinemachineDollyCart.cs

```diff
@@ -51,6 +54,15 @@ public enum UpdateMethod
         [FormerlySerializedAs("m_CurrentDistance")]
         public float m_Position;
 
+        private void OnValidate()
```

Comment:
OnValidate is called after changes in the inspector, and after an object is deserialized.  If some user script changes m_path during runtime, then OnValidate won't be called, so you need to be validating this more frequently.  Since the cost of this test is small, it doesn't hurt to validate on each use.

Reply:
I remove `OnValidate` function and move this part of code into `Update()`. Please LMK if there is a better place to place it